config.json

```diff
@@ -53,6 +53,14 @@
         "practices": [],
         "prerequisites": [],
         "difficulty": 2
+      },
+      {
+        "slug": "forth",
+        "name": "Forth",
+        "uuid": "dafa6994-946a-462e-9107-25becff892ce",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 1
```

Comment:
I would suggest bumping the difficulty by a lot. Probably 7 or 8 (7-9 map to "difficult")